instance.tf

```diff
@@ -14,10 +14,11 @@ resource "aws_security_group" "allow_ssh" {
     vpc_id = aws_vpc.core-infra.id
 
     ingress {
-      description = "TLS from VPC"
+      description = "allow ssh through tcp protocol"
```

Comment:
Nope, it's redundant. SSH uses tcp protocol. It's not like your code is allowing ssh through tcp protocol.